frontend/js/tests/fresh-releases/FreshReleases.test.tsx

```diff
@@ -73,7 +66,7 @@ describe("FreshReleases", () => {
     mountOptions.context.APIService.fetchSitewideFreshReleases = mockFetchSitewideFreshReleases;
     const wrapper = mount(
       <GlobalAppContext.Provider value={{ ...mountOptions.context }}>
-        <FreshReleases {...props} />
+        <FreshReleases />
```

Comment:
```suggestion
        <FreshReleases {...freshReleasesProps} />
```

Reply:
In the FreshReleases.tsx file the props only consisted of newAlert prop which is no more there. Hence the test failed for me with  `error TS2559: Type '{ user: { name: string; id: number; }; profileUrl: string; spotify: { access_token: string; permission: string[]; }; youtube: { api_key: string; }; }' has no properties in common with type 'IntrinsicAttributes'.` Please suggest me how to proceed.

Reply:
My bad, this looks fine without the props 👍